server/server.go

```diff
@@ -190,22 +167,56 @@ func (sc *ServerConfig) LoadFromFile(file string) error {
     sc.HistoryForwardThreshold = ysc.History.ForwardThreshold
     sc.AlertsForwardInterval = ysc.Alerts.ForwardInterval
     
-    clientCertX509, _ := x509.ParseCertificate(clientCertificate.Certificate[0])
-    serverCertX509, _ := x509.ParseCertificate(serverCertificate.Certificate[0])
-    clientCN := clientCertX509.Subject.CommonName
-    serverCN := serverCertX509.Subject.CommonName
-    
-    if len(clientCN) == 0 {
-        return errors.New("The common name in the certificate is empty. The node ID must not be empty")
-    }
-     
-    if clientCN != serverCN {
-        return errors.New(fmt.Sprintf("Server and client certificates have differing common names(%s and %s). This is the string used to uniquely identify the node.", serverCN, clientCN))
+    if (YAMLTLSFiles{}) != ysc.TLS  {
```

Comment:
These changes seem beyond the scope of getting devicedb to talk to the cloud through fp-edge.

Reply:
These changes are required. It may look messy because I had to move things around. But all I am doing is checking if TLS section is provided or not and getting nodeid if no certificates are provided.

Reply:
we need to cover the case where TLS section is empty in the config file, otherwise we will get errors.